dags/ssl_ratios.py

```diff
@@ -19,9 +19,9 @@
 with DAG("ssl_ratios", default_args=default_args, schedule_interval="@daily") as dag:
     # most downstream dependency is search_clients_daily
     wait_for_main_summary = ExternalTaskSensor(
-        task_id="wait_for_main_summary",
+        task_id="wait_for_copy_deduplicate",
```

Comment:
```suggestion
        task_id="wait_for_copy_deduplicate_main_ping",
```

Just to be explicit.